packages/runtime-common/realm.ts

```diff
@@ -967,8 +967,10 @@ export class Realm {
   ) {
     // If the realm is public readable or writable, do not require a JWT
     if (
+      request.url.endsWith('_message') ||
```

Comment:
Let's add some comments here explaining why we are bypassing request types in this condition

Reply:
Maybe we can organize these kinds of requests into a structure like this

```
endpointsWithoutAuthNeeded = [
    { path: '/_session', method: 'POST' },
    { path: '/_message', method: ... }, ...
  ];
 ```

Reply:
updated [c431eb3](https://github.com/cardstack/boxel/pull/1053/commits/c431eb32d32dd2d4300bc27115ab81d01d1ccea3)